src/ipc/handler.c

```diff
@@ -719,12 +719,18 @@ static int ipc_dma_trace_config(uint32_t header)
 	struct dma_sg_elem_array elem_array;
 	uint32_t ring_size;
 #endif
-	struct sof_ipc_dma_trace_params params;
```

Comment:
Can you confirm that the deprecated method will still work ? (albeit with no timestamp)

Reply:
@lgirdwood Yes, I tried with older kernel and it works.